pkg/manager/manager.go

```diff
@@ -216,6 +231,20 @@ func (m *manager) ResetVFConfig(conf *types.PluginConf) error {
 		}
 	}
 
+	// Restore MTU
+	if conf.MTU != 0 {
```

Comment:
should be in ReleaseVF.

in general SetupVF and ReleaseVF deal with the VF netdev itself.

ApplyVFConfig / ResetVFConfig deal with VF configuration on PF (ip link set <pf> vf <idx> )

AttachRepresentor / DetachRepresentor deal with VF rep configuration